apps/site/src/lib/blocks.ts

```diff
@@ -313,15 +313,6 @@ export const retrieveBlockReadme = async (
   blockMetadata: ExpandedBlockMetadata,
```

Comment:
@kachkaev to note that there are still some `readFileSync` /`readFile` calls above that can be removed when we remove locally-built blocks

Reply:
Thanks! Removed in 94c0630e6446a703cb7ec53b3a2245be75d98556